CONTRIBUTING.md

```diff
@@ -7,6 +7,101 @@
   * [Suggesting Enhancements](#suggesting-enhancements)
   * [Pull Requests](#pull-requests)
```

Comment:
some links are dead, please verify - should we update TOC, or remove it?

Reply:
I would suggest removing TOC. It's not really required for a doc like this.

Reply:
removed TOC